recipe/meta.yaml

```diff
@@ -55,23 +55,25 @@ requirements:
         - geos
         - proj.4
         - xerces-c
-        - libnetcdf
+        - libnetcdf 4.3*
         - krb5
-        - libtiff
         - libpng
-        - openjpeg
+        - openjpeg  # [not py35]
+        - libtiff 4.0.6  # [not py35]
```

Comment:
> and that is not working on OS X.

In what way? Is the pinned version wrong, or is the defaults package wrong? If it is the latter, we can get it fixed. I don't want to go down the route of doing something "because it worked" and would much sooner be representing this *accurately* even if it doesn't work. That way, we can just fix the things that don't work (in this case libtiff) and 🎉